plugin/trino-ranger/README.md

```diff
@@ -0,0 +1,95 @@
+       * $RANGER_HOME/usersync/install.properties
+         * In most enterprise environments, ActiveDirectory/LDAP are used to manage users and their roles.
+         * I do NOT recommend trying to manage users inside of Apache Ranger UI. I would configure the usersync service to pull the groups from AD/LDAP and attach your Ranger SQL policies to those external AD/LDAP groups.
+         * Most of the defaults work fine. The configs you want to pay attention to are
+           * SYNC_SOURCE=(ldap), rangerUsersync_password (configured above), SYNC_INTERVAL, SYNC_LDAP_URL, SYNC_LDAP_BIND_DN, SYNC_LDAP_BIND_PASSWORD, SYNC_LDAP_DELTASYNC, SYNC_LDAP_SEARCH_BASE, SYNC_LDAP_USER_SEARCH_BASE, SYNC_LDAP_USER_SEARCH_SCOPE, SYNC_LDAP_USER_OBJECT_CLASS, SYNC_LDAP_USER_SEARCH_FILTER, SYNC_LDAP_USER_NAME_ATTRIBUTE, SYNC_LDAP_USER_GROUP_NAME_ATTRIBUTE, SYNC_LDAP_USERNAME_CASE_CONVERSION, SYNC_LDAP_GROUPNAME_CASE_CONVERSION
+           * SYNC_INTERVAL specifies the time in minutes that you want Ranger to synchronize AD/LDAP. This can take some time so dont make it too short. 360 (6 hours) is a good number.
+     * With both the $RANGER_HOME/install.properties and $RANGER_HOME/usersync/install.properties configured run the setup scripts
+       * $RANGER_HOME/setup.sh
+       * $RANGER_HOME/usersync/setup.sh
+     * Now try and start the main ranger service.
+       * $RANGER_HOME/ews/start-ranger-admin.sh
+         * I highly recommend monitoring the ranger PID, /run/ranger/rangeradmin.pid, and restarting it of it should fail. If using Docker/kubernetes a simple bash script like so works well
+         ```
+         ranger_admin_pid=`cat /run/ranger/rangeradmin.pid` > /dev/null 2>&1
+
+         echo "${0##*/}:$LINENO: Waiting for ranger_admin_pid = $ranger_admin_pid"
+
+         while s=`ps -p $ranger_admin_pid -o s=` && [[ "$s" && "$s" != 'Z' ]]; do
+             sleep 1
+         done
+
+         echo "${0##*/}:$LINENO: Ranger admin service exited!!!"
+         ```
+
+     * Now try and start the AD/LDAP usersync serivce
+       * $RANGER_HOME/usersync/ranger-usersync-services.sh start
+       * NOTE: If you are running multiple instances of the Ranger UI, you should only ever have 1 and only 1 AD/LDAP usersync service running. Your groups will not properly sync otherwise.
+     * Now login to the ranger UI and configure the Trino service.
+       * Example:
+         * http://localhost:6080
+         * User: admin, Password: (defined above in rangerAdmin_password)
+     * Create your Trino service.
+       * The name is important and needs to be configured in the access-control-ranger.properties file. As of version 2.2.0, just use the Presto service type. Presto, also known as PrestoSQL, is the old name for Trino.
+       * The defult policies that Ranger installs under your above service is WIDE open. Everything works for everyone. I will not document the full ranger setup.
```

Comment:
We should document the full ranger setup somewhere in Trino land though and point to it.

Reply:
Where?